source/config.js

```diff
@@ -235,6 +239,25 @@ if (typeof currentRootPath !== 'string') {
 }
 module.exports.rootPath = currentRootPath;
 
+let currentGitCommand = module.exports.gitCommand;
+if (typeof currentGitCommand !== 'string') {
+  currentGitCommand = '';
+} else if (currentGitCommand !== '') {
+  // must start with a slash
+  if (currentGitCommand.charAt(0) !== '/') {
+    currentGitCommand = '/' + currentGitCommand;
+  }
+  // can not end with a trailing slash - it should be a command
+  if (currentGitCommand.charAt(currentGitCommand.length - 1) === '/') {
```

Comment:
instead of checking for this.  Can we literally concatenate "/git" at the end?  Also, it would be `gitPath` instead of `gitCommand`.  I would like to limit the possibility that ungit can execute non git commands.  

Furthermore, considering this is a crux of all operations, I would like to reject on invalid inputs rather than adjust on invalid input as well to make the intent explicit.